apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget.cs

```diff
@@ -491,7 +491,18 @@ private LogEntry BuildLogEntry(LogEventInfo loggingEvent)
             if (SendJsonPayload)
             {
                 var jsonStruct = new Struct();
-                jsonStruct.Fields.Add("message", Value.ForString(RenderLogEvent(Layout, loggingEvent)));
+                if (EnableJsonLayout)
```

Comment:
Just to check my understanding: the idea is that most of the struct (properties, service etc) will be populated as normal this allows additional fields to be set in the JSON struct, and the message field is not automatically emitted.

I think I've understood correctly but I wanted to check - partly as I expect to write up documentation for this at some point.

Reply:
Yes, but like @snakefoot and @amanda-tarafa said it'd be better to change that to overriding the whole structure